src/types/redis_list.cc

```diff
@@ -392,6 +392,67 @@ rocksdb::Status List::Range(const Slice &user_key, int start, int stop, std::vec
   return rocksdb::Status::OK();
 }
 
+rocksdb::Status List::Pos(const Slice &user_key, const Slice &elem, const PosSpec &spec,
+                          std::vector<int64_t> *indexes) {
+  assert(spec.rank != 0);
```

Comment:
I'm not sure whether we need those assertions because our CI and tests are using the `RelWithDebugInfo` build which doesn't see these checks.

Reply:
Yes, can remove this line.